inspirehep/modules/disambiguation/search.py

```diff
@@ -0,0 +1,366 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Inspire.
+# Copyright (C) 2016 CERN.
+#
+# Inspire is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Inspire is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Inspire; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+"""Set of methods to query Elasticsearch instance."""
+
+from __future__ import absolute_import, division, print_function, \
+    unicode_literals
+
+from dateutil.parser import parse
+
+from elasticsearch.helpers import scan
+from flask import current_app
+
+from invenio_search import current_search_client as es
+
+
+def _get_record(record_id, source):
+    """Make a call to the Elasticsearch instace with known record_id.
+
+    :param record_id:
+        An identification (UUID) of the given record.
+
+        Example:
+            record_id = "a5afb151-8f75-4e91-8dc1-05e7e8e8c0b8"
+
+    :param source:
+        A list of fields to be extracted from the Elasticsearch response.
+
+        Example:
+            source = ["collaboration", "author.full_name"]
+    """
+    elasticsearch_index = current_app.config.get('DISAMBIGUATION_RECORD_INDEX')
+    elasticsearch_type = current_app.config.get('DISAMBIGUATION_RECORD_TYPE')
+
+    return es.get_source(
+        index=elasticsearch_index,
+        id=str(record_id),
+        doc_type=elasticsearch_type,
+        _source=source)
```

Comment:
@jacenkow I believe here you should also add a FIXME, in case we are going to run disambiguation over multiple indexes (which is the case, i.e. Literature and Data)